### PR TITLE
Add an item limit to TimedStore

### PR DESCRIPTION
**Pull request by vmarmol**
Made the default limit 100,000 elements which is a little over 1 event per second for a day and assuming 250 bytes per event it would take up ~24MiB per event type.

/cc @rjnagal

### CODE REVIEW

File: manager/manager.go [hunk 1]
```diff
@@ -137,7 +137,7 @@ func New(memoryStorage *memory.InMemoryStorage, sysfs sysfs.SysFs) (Manager, err
 	glog.Infof("Version: %+v", newManager.versionInfo)
 
 	// TODO(vmarmol): Make configurable.
-	newManager.eventHandler = events.NewEventManager(24 * time.Hour)
+	newManager.eventHandler = events.NewEventManager(24*time.Hour, 100000)
```
Comment (rjnagal) [inline manager/manager.go]: Can we make this a flag?

Comment (vmarmol) [inline manager/manager.go]: Yes, the TODO is to make it configurable.

**Comment by rjnagal** [comment]
LGTM

File: manager/manager.go (hunk 1 above)
Comment (vmarmol) [inline manager/manager.go]: (since we'd still like to do the per-event limit before we expose it as a flag)

**Pull request merged**